docs/protocols/scp-1.md

```diff
@@ -0,0 +1,45 @@
+# SCP-1 (Token)
+
+## Definition
+
+SCP-1 is the most basic token standard. It allows to set a token name, a ticker, and a maximum supply. Upon creation, no tokens will be minted. The token creator can then use the
+mint command on the console. The mint command allows the creator to mint new tokens, provided this doesn’t increase the existing supply above the maximum set at the start. This kind of token standard can be employed in usecases that require certain degree of centralization, such as stablecoins or tokenized services like MineCube workers.
+SCP-1 tokens may also be burnt, which reduces the circulating supply without affecting the maximum.
+
+## Deploying Fee
+Deploying a SCP-1 token requires a static **10 SCC**, with transactions fees (~0.00002 SCC) additionally.
+
+This fee is irrecoverable 'burned'.  
```

Comment:
irrecoverabl**y**(?), otherwise it reads as `irrecoverable burned`, or we could use: `... irrecoverable. e.g; burned.`